File: test10.js
function isDigit(s) {
    //your code
    let num = Number(s)
    return(!isNaN(num))
}


isDigit("3")
isDigit("  3  ")
isDigit("-3.23")

isDigit("3-4")
isDigit("  3   5")
isDigit("3 5")
isDigit("zero")

File: test10.py
def is_digit(s):
    result = True
    try:
        num = float(s)
    except ValueError:
        result = False

    return result

# Test cases
print(is_digit("3"))
print(is_digit("  3  "))
print(is_digit("-3.23"))
print(is_digit("3-4"))
print(is_digit("  3   5"))
print(is_digit("3 5"))
print(is_digit("zero"))
